tests/e2e/performance/test_bulk_pod_attachtime_performance.py

```diff
@@ -220,6 +223,17 @@ def test_bulk_pod_attach_performance(self, teardown_factory, bulk_size):
 
         full_results.add_key("storageclass", self.sc)
         full_results.add_key("pod_bulk_attach_time", bulk_total_time)
+        full_results.add_key("pvc_size", self.pvc_size)
+        full_results.add_key("interface", self.interface)
```

Comment:
since you gave the 'storageclass' you do no need this, it is duplicate.

Reply:
Done.